flink-table/flink-table-runtime/pom.xml

```diff
@@ -137,7 +137,7 @@ under the License.
 			<version>${project.version}</version>
 			<scope>test</scope>
 		</dependency>
-    </dependencies>
+	</dependencies>
```

Comment:
can we remove it from change log?

Reply:
Aren't we squashing all commits? It'll then not be in the commit/changelog, since I removed the change

Or do you think it'd be better to merge a a couple of separate commits and not squash? I could then rebuild the commits when we're done with the reviews